Resources/public/js/views/services/plugins/ez-contenteditplugin.js

```diff
@@ -34,7 +34,7 @@ YUI.add('ez-contenteditplugin', function (Y) {
          * @method _checkExistingDraft
          * @protected
          * @param {Object} e event facade of the editContentRequest event
-         * @param {eZ.Content} e.content being edited
+         * @param {eZ.Content|eZ.ContentInfo} e.content being edited
```

Comment:
it's the counter part of https://github.com/ezsystems/PlatformUIBundle/pull/673/files#r78139237 this method should be able to accept either the Content or Content Info but in 2 different property of the event facade.